<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Status</class>
 <widget class="QDialog" name="Status">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>376</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Status</string>
  </property>
  <widget class="QCheckBox" name="updateBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>92</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="scaleBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>92</y>
     <width>98</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>160</y>
     <width>250</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="colLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="rowLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="minLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="y0Label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="x0Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="dyLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="dxLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>16</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>244</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="plusButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qtoma2.qrc">
        <normaloff>:/Resources/Resources/plus.jpg</normaloff>:/Resources/Resources/plus.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="boxButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qtoma2.qrc">
        <normaloff>:/Resources/Resources/box.jpg</normaloff>:/Resources/Resources/box.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="cboxButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qtoma2.qrc">
        <normaloff>:/Resources/Resources/c-box.jpg</normaloff>:/Resources/Resources/c-box.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="rulerButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qtoma2.qrc">
        <normaloff>:/Resources/Resources/ruler.jpg</normaloff>:/Resources/Resources/ruler.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="graphButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qtoma2.qrc">
        <normaloff>:/Resources/Resources/curve.jpg</normaloff>:/Resources/Resources/curve.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>250</y>
     <width>254</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="xLabel_1">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="yLabel_1">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="zLabel_1">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Z: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="xLabel_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="yLabel_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="zLabel_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Z:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>220</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>81</height>
       </size>
      </property>
      <property name="title">
       <string>Color Min</string>
      </property>
      <widget class="QPushButton" name="cminMinus">
       <property name="geometry">
        <rect>
         <x>-5</x>
         <y>47</y>
         <width>50</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cminPlus">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>47</y>
         <width>53</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QLabel" name="colorMinValue">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>21</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>cminvalue</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>81</height>
       </size>
      </property>
      <property name="title">
       <string>Color Max</string>
      </property>
      <widget class="QPushButton" name="cmaxMinus">
       <property name="geometry">
        <rect>
         <x>-5</x>
         <y>47</y>
         <width>50</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cmaxPlus">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>47</y>
         <width>53</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QLabel" name="colorMaxValue">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>21</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>cmaxvalue</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>116</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QLabel" name="cMinMaxIncLabel">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Min/Max Increment: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollBar" name="cMinMaxIncControl">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="qtoma2.qrc"/>
 </resources>
 <connections/>
</ui>
